### ISSUE TITLE
Pconv Multi-scale training AP is zero.

### ISSUE THREAD

**Issue by tingtingliangvs** [jshilong/SEPC#3]
Hi there, thanks for sharing the code, the work is very inspiring!
I have a problem when using Pconv in FPN. When I train in single scale, the result is quite normal, but when I train in multi-scale, for example, set following [HTC config](https://github.com/jshilong/SEPC/blob/d18e83ab9472fbe1e4ac4103d1abdbb0661da268/mmdetection/configs/htc/htc_dconv_c3-c5_mstrain_400_1400_x101_64x4d_fpn_20e.py) dict(
        type='Resize',
        img_scale=[(1600, 400), (1600, 1400)],
        multiscale_mode='range',
        keep_ratio=True),
the evaluation result is 0. Would you please share the multiscale training config?

**Comment by jshilong** [comment]
Thanks！The code released is a little different with first version (some key in checkpoint is different), So I have to prepare 2x model again,  It will take some time :(.

You can check the loss in the log ( has Nan or other unusual values) or upload the complete config,  I will help you check it.

**Comment by tingtingliangvs** [comment]
It turned out that I didn't add "norm_cfg=dict(type='BN', requires_grad=True)" in backbone config. Problem solved! Thanks a lot! By the way, could you please share the size of the multi-scale training input range?

**Comment by jshilong** [comment]
all about our 2X results setting is provided in our paper,  640\~800 for all 2x results except for dcn backbone(we use 480\~960 for this model and get 50.1 mAP with single scale test(800)) on test dataset)
